db.go

```diff
@@ -403,7 +403,7 @@ func (db *DB) GetPropertyCF(propName string, cf *ColumnFamilyHandle) string {
 	cProp := C.CString(propName)
 	defer C.free(unsafe.Pointer(cProp))
 	cValue := C.rocksdb_property_value_cf(db.c, cf.c, cProp)
-	defer C.free(unsafe.Pointer(cValue))
```

Comment:
This change is unlike all the rest.  Every other change is an error string.  Can you explain?

Reply:
According to the link I posted all memory coming from rocksdb should be freed with `rocksdb_free`. So this still seems reasonable. If anything when I grep the repo, even more `C.free` calls should be changed.

Reply:
Confirmed: This memory is allocated by strdup and should be freed as well